backend/controllers/bulkAppointmentsClashes.js

```diff
@@ -82,6 +82,17 @@ const bulkAppointmentsClashesFactory = (elite2Api, logError) => {
     } = req.session
 
     const remainingPrisoners = prisonersListed.filter(prisoner => !req.body[prisoner.offenderNo])
+    const removedPrisoners = prisonersListed.filter(prisoner => req.body[prisoner.offenderNo])
```

Comment:
```suggestion
    const removedPrisoners = prisonersListed.filter(prisoner => req.body[prisoner.offenderNo]).map(prisoner => ({
      offenderNo: prisoner.offenderNo,
      lastName: prisoner.lastName,
    }))
```

Don't need the separate `prisonersRemoved` then

Reply:
or use a .reduce, probably better.